daemon/access.go

```diff
@@ -246,3 +249,48 @@ func (ac interfaceAuthenticatedAccess) CheckAccess(d *Daemon, r *http.Request, u
 
 	return Unauthorized("access denied")
 }
+
+// noticesReadAccess allows mapping notice types to access checkers and otherwise
+// behaves like openAccess (ie. allows access from snapd.socket only).
```

Comment:
the last part of the comment is not quite true, as we a type is not listed at all the effect is to deny access?